templates/member/navConnection.twig

```diff
@@ -0,0 +1,15 @@
+{% if connected %}
```

Comment:
https://symfony.com/doc/current/security.html#fetch-the-user-in-a-template

Reply:
Ok, I will change that with a member system branch.